sway/sway.5.scd

```diff
@@ -558,6 +558,13 @@ The default colors are:
 	Set the opacity of the window between 0 (completely transparent) and 1
 	(completely opaque).
 
+*tiling\_drag*  enable|disable|toggle
+	Sets whether or not tiling containers can be dragged with the mouse. If
+	enabled (default), the _floating_mod_ can be used to drag tiling, as well
```

Comment:
Doesn't the middle underscore need to be escaped? Also again two lines down.

Reply:
Oops. Yes, they do. Fixed.